polls/templates/polls/results.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'global/skeleton.html' %}
+
+{% block pagename %}Visualizza i risultati{% endblock pagename %}
+
+{% block content %}
+
+    <div class="d-flex justify-content-center">
+        <div class="card text-center" >
+            <div class="card-body">
```

Comment:
Big note on template: of course, graphical part needs improvement and it's okay.

Try applying those suggestions:
- Take `vote_confirm.html` template as an example. Try replicate that kind of structure instead of the home page's one (that is a very particular thing...). In practice...
- instead of one big card, keep the "two main separated components" structure (one from the title, one for the proper content). You may copy the ones used on `vote_confirm.html` template
- **Very important note:** **don't use dummy dummy texts like** _"Questa è la pagina dei risultati"_, but try instead since the beginning "putting value" and faking a real page. That will help you entering in the mindset of producing good valuable html it shouldn't be rewritten at all